lib/maze/api/appium/file_manager.rb

```diff
@@ -45,7 +61,15 @@ def read_app_file(filename, directory = nil)
           end
 
           $logger.trace "Attempting to read file from '#{path}'"
-          file = @driver.pull_file(path)
+          @driver.pull_file(path)
+          true
```

Comment:
Since this is attempting to retrieve a file this should be the file pulled rather than the success metric?

Reply:
Gosh, you're absolutely right - I'd assumed the file is written straight to file, but it is returned in memory by the method call.  Good spot.